mlos_bench/mlos_bench/launcher.py

```diff
@@ -203,9 +205,14 @@ def _parse_args(parser: argparse.ArgumentParser, argv: Optional[List[str]]) -> T
                  ' a single trial with default (or specified in --tunable_values).')
 
         parser.add_argument(
-            '--trial_config_repeat_count', '--trial-config-repeat-count', required=False, type=int, default=1,
```

Comment:
Dropping `default=1` because now the default can be enforced in the chosen `Scheduler` class?

Reply:
that, too - but the main reason is that since command-line options have priority over JSON config settings, having default value here means that we always override values in configs with 1 when command-line option is not specified :)

Reply:
Ah, makes sense.  Thanks!